### DISCUSSION TITLE
Create Project for Team Review / Collaborate with multiple researchers

### DISCUSSION THREAD

**Discussion by acodom** [asreview/asreview#909] [Q&A] [answered]
Hello! I am new to ASReview. After following through the documentation I could find to do my own individual set up, I'm trying to figure out if it is possible to work as a team on ASReview. There are several researchers working in different countries on the same review, and we would like to each do some of the screening. Is there a way to have joint access to the same project and to work together remotely to finish screenings? Could you point me towards a help page / discussion if it already exists?

**Comment by valmelnikov** [comment, marked as answer]
Hey @acodom , in [MegaMeta](https://github.com/asreview/paper-megameta-postprocessing-screeningresults) study we set up ASReview on the server to make it possible for multiple people to work on the same ASReview project(s). We used [`http-auth` branch](https://github.com/valmelnikov/asreview/tree/http-auth) of ASReview, which enables authentication of the users and project-locking by a single user to ensure that no project is reviewed by multiple people at the same time. The server setup is described in [this repo](https://github.com/valmelnikov/asreview_server_setup). As I was responsible for that setup, I am happy to give more clarifications on it if needed.

**Reply by acodom** [reply]
Hi @valmelnikov - is it necessary to set up a server to allow multiple people to have access to work on the same project? Are there other ways to do this? 
Looking through your repo - I am thinking we will have trouble with getting login access for two of our researchers, as they are at a different, international institution. Any advice?

**Reply by valmelnikov** [reply]
In that particular setup the access was extended to IP addresses of external institutions (AMC) the researchers were coming from. So, in principle, this is not a big problem. Honestly, I cannot imagine a robust solution for collaborative work without a central server and access control through authentication.

**Reply by acodom** [reply]
Thanks @valmelnikov! I will look into our options and ask if we need more clarification for the server option.
